.drone.star

```diff
@@ -1292,7 +1386,6 @@ def redisService():
 		'name': 'redis',
 		'image': 'webhippie/redis',
 		'pull': 'always',
-		'detach': True,
```

Comment:
what is this `detach` for:question:

Reply:
We are now running redis as a service not step so we can remove that.